xbmc/platform/darwin/ios-common/DarwinEmbedNowPlayingInfoManager.h

```diff
@@ -0,0 +1,32 @@
+/*
+ *  Copyright (C) 2010-2019 Team Kodi
+ *  This file is part of Kodi - https://kodi.tv
+ *
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *  See LICENSES/README.md for more information.
+ */
+
+#import <Foundation/Foundation.h>
+
+typedef NS_ENUM(unsigned int, DarwinEmbedPlaybackState) {
+  DARWINEMBED_PLAYBACK_STOPPED,
+  DARWINEMBED_PLAYBACK_PAUSED,
+  DARWINEMBED_PLAYBACK_PLAYING
+};
+
+@interface DarwinEmbedNowPlayingInfoManager : NSObject <NSCopying>
```

Comment:
our class doesn't have to implement `NSCopying`. Sorry if my previous comment was unclear.